places/app/views/users/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_for @user do |f| %>
+    
+    <div>
+        <%= f.text_field :first_name, placeholder: 'First Name' %>
+    </div>
+    <div>
+        <%= f.text_field :last_name, placeholder: 'Last Name' %>
+    </div>
+    <div>
+        <%= f.text_field :email, placeholder: 'Email' %>
+    </div>
+    <div>
+        <%= f.text_field :picture, placeholder: 'Picture' %>
+    </div>
+    <div>
```

Comment:
Your model associations are setup correctly, but i don't see how you are assigning users to places. How about adding a select tag here to allow you to choose from the list of available places? Check out the [docs](https://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-select_tag) for more info on select tag.